examples/utxo/simpleapp/src/main/java/io/horizen/examples/SimpleAppModule.java

```diff
@@ -174,8 +173,5 @@ public void configureApp() {
         bind(Integer.class)
```

Comment:
Can we add a python test like **sc_bt_limit_across_epoch** to test the SC2SC fork activation?